content/hi/cloud-computing.md

```diff
@@ -0,0 +1,26 @@
+---
+title: क्लाउड कंप्यूटिंग (Cloud Computing)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+क्लाउड कंप्यूटिंग एक ऐसा मॉडल है जो इंटरनेट पर मांग पर सीपीयू, नेटवर्क और डिस्क क्षमताओं जैसे कंप्यूट संसाधनों की पेशकश करता है।
+क्लाउड कंप्यूटिंग उपयोगकर्ताओं को दूरस्थ भौतिक स्थान में कंप्यूटिंग शक्ति तक पहुँचने और उपयोग करने की क्षमता देती है।
```

Comment:
since in the previous statement, we've given masculine attribute to cloud computing - `संसाधनों की पेशकश करता है।` , let's continue that for the rest of the content. 
e.g. उपयोग करने की क्षमता देता है।